cmdeploy/src/cmdeploy/__init__.py

```diff
@@ -441,6 +441,44 @@ def check_config(config):
     return config
 
 
+def deploy_mtail(config):
+    apt.packages(
+        name="Install mtail",
+        packages=["mtail"],
+    )
+
+    # Using our own systemd unit instead of `/usr/lib/systemd/system/mtail.service`.
+    # This allows to read from journalctl instead of log files.
+    files.template(
+        src=importlib.resources.files(__package__).joinpath("mtail/mtail.service.j2"),
+        dest="/etc/systemd/system/mtail.service",
+        user="root",
+        group="root",
+        mode="644",
+        address="127.0.0.1",  # TODO read from config
```

Comment:
i think this needs to happen still in this PR -- if the chatmail.ini does not configure mtail address + port, then we should not setup or at least not start mtail i guess.